source/frontend/src/hooks/repositories/mapLayer/useHighwayLayer.tsx

```diff
@@ -0,0 +1,58 @@
+import { FeatureCollection, Geometry } from 'geojson';
+import { LatLngLiteral } from 'leaflet';
+import { useCallback, useContext, useMemo } from 'react';
+
+import { useLayerQuery } from '@/hooks/layer-api/useLayerQuery';
+import { ISS_ProvincialPublicHighway } from '@/models/layers/pimsHighwayLayer';
+import { TenantContext } from '@/tenants';
+
+/**
+ * API wrapper to centralize all AJAX requests to WFS endpoints for the pims property location.
+ * @returns Object containing functions to make requests to the WFS layer.
+ * Note: according to the view ISS_ProvincialPublicHighway
```

Comment:
"...WFS endpoints for the **pims property location**" -> "... for the highway layer locations"